src/axom/quest/tests/quest_sampling_shaper.cpp

```diff
@@ -1509,6 +1720,91 @@ dimensions: 2
   }
 }
 
+TEST_F(SamplingShaperTest3D, contour_and_stl_3D)
```

Comment:
This test uses a 3D background mesh, and shapes in c2c contours using a projection function, as well as 3D stl meshes.